src/recorder.js

```diff
@@ -155,16 +155,19 @@ Recorder.prototype.pause = function( flush ) {
   if ( this.state === "recording" ) {
     this.state = "paused";
     if ( flush && this.config.streamPages ) {
+      var encoder = this.encoder;
```

Comment:
Is this an optimization?

Reply:
A following PR will delete the encoder when it's destroyed, so it's a safety feature (possibly unnecessary since it's hard to see how this would overlap).